### PR TITLE
Column description for the Return page

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request adds column description for the Return page

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/sales/returns.html

### CODE REVIEW

File: src/sales/returns.md [hunk 1]
```diff
@@ -13,6 +13,19 @@ _Returns grid_
...
 RMAs can be issued for simple, grouped, configurable, and bundle product types. However, RMAs are not available for virtual products, downloadable products, and gift cards.
 
+### Column descriptions
+
+|Column|Description|
+|--- |--- |
+|Select|Select the checkbox to select the return(s) to be subject to an action, or use the selection control in the column header. Options: Select All / Deselect All / Select Visible / Unselect Visible|
+|RMA|A unique numeric identifier that is assigned to each return|
+|Requested|Date and time the return was placed|
+|Order|Unique serial number of the returned order|
```
Comment (dobooth) [inline src/sales/returns.md:23]: This isn't really a serial number per se. This is the original order number, correct?

**Review by dobooth — CHANGES_REQUESTED** [review]

File: src/sales/returns.md (hunk 1 above)
Comment (ZhovnirMaria) [reply src/sales/returns.md]: Yes, you are right, @dobooth 
I changed it.

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the pull request successfully imported.

**Pull request merged**